src/main/java/no/unit/nva/cristin/projects/model/nva/EmptyNvaProject.java

```diff
@@ -0,0 +1,12 @@
+package no.unit.nva.cristin.projects.model.nva;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(NON_NULL)
+public class EmptyNvaProject extends NvaProject {
```

Comment:
Not sure about this…should probably have a discussion (I probably don't understand)